owl/owl.rdf

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0"?>
+<!DOCTYPE owl [
+
+     <!ENTITY rdf  "http://www.w3.org/1999/02/22-rdf-syntax-ns#" >
+     <!ENTITY rdfs "http://www.w3.org/2000/01/rdf-schema#" >
+     <!ENTITY xsd  "http://www.w3.org/2001/XMLSchema#" >
+     <!ENTITY owl  "http://www.w3.org/2002/07/owl#" >
+   ]>
+
+<rdf:RDF
+  xmlns     ="&owl;"
+  xmlns:owl ="&owl;"
+  xml:base  ="http://www.w3.org/2002/07/owl"
+  xmlns:rdf ="&rdf;"
+  xmlns:rdfs="&rdfs;"
+  xmlns:dc  ="http://purl.org/dc/elements/1.1/"
+>
+
+<Ontology rdf:about="">
+  <versionInfo>March 18. 2003, GS</versionInfo>
+  <imports rdf:resource="package://knowrob/owl/rdf-schema.xml"/>
```

Comment:
Is this really needed? I would prefer to not have this ros-specific import in here.